9jaswag/PostIt
# #152687636 Review Code Quality

#### What does this PR do?
- implements all code quality feedback

#### Description of Task to be completed?
- rename ```Searchpage.jsx``` to ```SearchPage.jsx```
- change the status code returned by ```findUser``` method in ```users.js```
- remove all commented code

#### How should this be manually tested?
- check the ```SearchPage.jsx``` to see the new naming in PascalCase
- check ```server/controller/users.js``` to see the new status code returned by ```findUser``` method
- Check all files to see there are no commented code

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
- #152687636

#### Screenshots (if appropriate)
- None

#### Questions:
- None
